### DISCUSSION TITLE
FreeBSD Install Guide

### DISCUSSION THREAD

**Discussion by verkaro** [charmbracelet/soft-serve#436] [Ideas]
Is it possible that you could have a guide to installing and setting up a service for FreeBSD, like there is for Linux and Systemd? That would be kindly received.

**Comment by bw9ubwo** [comment]
Maybe I am too late here, but there is a ready to use package.

```
pkg install soft-serve

# /etc/rc.conf
soft_serve_enable="YES"
soft_serve_initial_admin_key=""
soft_serve_port="22"
soft_serve_host="domain.tld"
```
